_inc/client/components/masthead/index.jsx

```diff
@@ -35,7 +35,7 @@ const Masthead = React.createClass( {
 							</a>
 						</li>
 						<li className="jp-masthead__link-li">
-							<a href="http://surveys.jetpack.me/research-plugin?rel=3.9.4" target="_blank" className="jp-masthead__link">
+							<a href="http://surveys.jetpack.me/research-plugin?rel=4.3.0" target="_blank" className="jp-masthead__link">
```

Comment:
Should we make this a dynamic value, so we don't have to update it manually for each release?